packages/server/api/src/app/database/migrations/1748356358052-AddTemplateCategories.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddTemplateCategories1748356358052 implements MigrationInterface {
+  name = 'AddTemplateCategories1748356358052';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "flow_template"
+            ADD "categories" jsonb
+        `);
+  }
+
+  public async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            ALTER TABLE "flow_template" DROP COLUMN "categories"
+        `);
+  }
```

Comment:
We are not using it.

```suggestion
  public async down(queryRunner: QueryRunner): Promise<void> {
    throw new Error('Rollback not implemented');
  }
```

Reply:
would not hurt to have it if someone changes to an older version, but I'll stay consistent then.